src/engine/sysDef.cpp

```diff
@@ -1749,6 +1749,20 @@ void DivEngine::registerSystems() {
     }
   );
 
+    sysDefs[DIV_SYSTEM_MSM5205]=new DivSysDef(
+    "OKI MSM5205", NULL, 0xab, 0, 1, false, true, 0x150, false, 1U<<DIV_SAMPLE_DEPTH_VOX, 0, 0,
```

Comment:
Use system IDs 0xfe or 0xff during development.
Once I allocate a new system ID, use that.

Reply:
I might do that, but when we release the 0.63. I might try again and add the sound chip again.

Reply:
explain me like i'm 5 why tf  would you want to? isnt adding it once enough?

Reply:
I'm not sure if I can make another sound chip, but if we want to do the CPS3 Support, we can do that, if I wanted to stop coding, then adding it once is enough for it.

Reply:
I might wait there, I feel like it might get approved, or not.

Reply:
What are you doing? You're not supposed to wait.

Reply:
? Shall we change all of these as soon as possible and we can approve it?

Reply:
Of course! If I request changes, you should fulfill these changes. Then I will review it again and so on until I approve and merge it.

Reply:
I'm no expert in C and C++ programming, but I can try my best to move it to the end of it and change some of the code.